### ISSUE TITLE
Error for code snippet 8.8 - NotImplementedError

### ISSUE THREAD

**Issue by EAly** [pymc-devs/pymc-resources#210]
Hi,

I get the following error when attempting to run the code snippet 8.8 (I'm using the environment in the attached txt file) : 

```
NotImplementedError                       Traceback (most recent call last)
/home/eaunix/xFiles/statsRethinking/ch008.ipynb Cell 27 in <cell line: 2>()
      [3](vscode-notebook-cell://wsl%2Bubuntu/home/eaunix/xFiles/statsRethinking/ch008.ipynb#X34sdnNjb2RlLXJlbW90ZQ%3D%3D?line=2) a = pm.Normal("a", 1, 0.1, shape=cid.categories.size)
      [4](vscode-notebook-cell://wsl%2Bubuntu/home/eaunix/xFiles/statsRethinking/ch008.ipynb#X34sdnNjb2RlLXJlbW90ZQ%3D%3D?line=3) b = pm.Normal("b", 0, 0.3)
----> [6](vscode-notebook-cell://wsl%2Bubuntu/home/eaunix/xFiles/statsRethinking/ch008.ipynb#X34sdnNjb2RlLXJlbW90ZQ%3D%3D?line=5) mu = a[cid] + b * (dd["rugged_std"] - 0.215)
      [7](vscode-notebook-cell://wsl%2Bubuntu/home/eaunix/xFiles/statsRethinking/ch008.ipynb#X34sdnNjb2RlLXJlbW90ZQ%3D%3D?line=6) sigma = pm.Exponential("sigma", 1)
      [9](vscode-notebook-cell://wsl%2Bubuntu/home/eaunix/xFiles/statsRethinking/ch008.ipynb#X34sdnNjb2RlLXJlbW90ZQ%3D%3D?line=8) log_gdp_std = pm.Normal("log_gdp_std", mu, sigma, observed=dd["log_gdp_std"])

File ~/miniconda3/envs/StReth/lib/python3.10/site-packages/aesara/tensor/var.py:519, in _tensor_py_operators.__getitem__(self, args)
    511     return (isinstance(val, (tuple, list)) and len(val) == 0) or (
    512         isinstance(val, np.ndarray) and val.size == 0
    513     )
    515 # Force input to be int64 datatype if input is an empty list or tuple
    516 # Else leave it as is if it is a real number
    517 # Convert python literals to aesara constants
    518 args = tuple(
--> 519     [
    520         at.subtensor.as_index_constant(
    521             np.array(inp, dtype=np.int64) if is_empty_array(inp) else inp
    522         )
    523         for inp in args
    524     ]
    525 )
    527 # Determine if advanced indexing is needed or not.  The logic is
...
---> 49     raise NotImplementedError(f"Cannot convert {x} to a tensor variable.")

NotImplementedError: Cannot convert [1, 0, 0, 0, 0, ..., 0, 0, 1, 1, 1]
Length: 170
Categories (2, int64): [0, 1] to a tensor variable.
```
[stats_reth2_ea_envExport.txt](https://github.com/pymc-devs/pymc-resources/files/9492612/stats_reth2_ea_envExport.txt)

**Comment by EAly** [comment]
Hi @yahrMason ,

Can you help with error?

**Comment by yahrMason** [comment]
@EAly I committed updates to the Chp_08 notebook on my most recent PR yahrMason:Chp_07_v4_update. I remember coming across this error when I was going through it. It should be fixed if you use the 8.8 snippet in the updated notebook.

**Comment by EAly** [comment]
Thanks a lot
